src/gc/gccommon.cpp

```diff
@@ -15,10 +15,10 @@
 #include "gc.h"
 
 #ifdef FEATURE_SVR_GC
-SVAL_IMPL_INIT(uint32_t,GCHeap,gcHeapType,GCHeap::GC_HEAP_INVALID);
+SVAL_IMPL_INIT(uint32_t,IGCHeap,gcHeapType,IGCHeap::GC_HEAP_INVALID);
 #endif // FEATURE_SVR_GC
 
-GPTR_IMPL(GCHeap,g_pGCHeap);
+IGCHeapInternal* g_theGcHeap;
```

Comment:
It looks like there are other things broken with the debugger - I see that the debugger makes assumptions about the location of `IGCHeap::gcHeapType